docs/multi_process.md

```diff
@@ -28,12 +28,15 @@ Key concepts:
   * Each process has a
     distinct set of *local* devices it can address. The *global* devices are the set
     of all devices across all processes.
-  * Use standard JAX parallelism APIs like {func}`~jax.pmap` and
-    {func}`~jax.experimental.maps.xmap`. Each process “sees” *local* input and
-    output to parallelized functions, but communication inside the computations
-    is *global*.
+  * Use standard JAX parallelism APIs like {func}`~jax.jit` (see
+    {doc}`/sharded-computation` tutorial) and
+    {func}`~jax.experimental.shard_map.shard_map`. Each process “sees”
+    *local* input and output to parallelized functions, but
+    communication inside the computations is *global*.
   * Make sure all processes run the same parallel computations in the same
     order.
+  * Make sure all processes has the same number of local devices and
```

Comment:
What do you mean by "that they are exactly the same"?

Reply:
I was thinking you can't have one node with 8 V100 and another one with 8 H100.
But in fact, you probably can. It will just run at V100 speed.
What isn't supported is different kind of devices inside the same process.

Can you confirm that each process must have the same number of devices?

Reply:
I tried to clarify all that.